### PR TITLE
Add support to checkout internal repo when internal PR testing

### PR DESCRIPTION
**Pull request by lerry-lee**
### Description of PR

Add support to checkout internal repo when internal PR testing. In PR #6859 , we add azp_access_token when download internal image. In this PR, we add azp_repo_access_token to access internal repo.

Signed-off-by: Chun'ang Li <chunangli@microsoft.com>

### Type of change

- [ ] Bug fix
- [ ] Testbed and Framework(new/improvement)
- [ ] Test case(new/improvement)

### Back port request
- [ ] 201911
- [ ] 202012
- [ ] 202205

#### What is the motivation for this PR?

Add support to checkout internal repo when internal PR testing. In PR #6859 , we add azp_access_token when download internal image. In this PR, we add azp_repo_access_token to access internal repo.

### CODE REVIEW

File: .azure-pipelines/test_plan.py [hunk 1]
```diff
@@ -331,12 +332,20 @@ def poll(self, test_plan_id, interval=60, timeout=36000, expected_states=""):
...
     parser_create.add_argument(
-        "--access-token",
+        "--azp-access-token",
         type=str,
-        dest="access_token",
+        dest="azp_access_token",
         default="",
         required=False,
-        help="Authorization token to access internal resource (image, etc)"
+        help="A special variable that carries the security token used by the running build"
```
Comment (yejianquan) [inline .azure-pipelines/test_plan.py:340]: Recommend: Token to download the artifacts of Azure Pipelines

File: .azure-pipelines/test_plan.py [hunk 2]
```diff
@@ -331,12 +332,20 @@ def poll(self, test_plan_id, interval=60, timeout=36000, expected_states=""):
...
+    )
+    parser_create.add_argument(
+        "--azp-repo-access-token",
+        type=str,
+        dest="azp_repo_access_token",
+        default="",
+        required=False,
+        help="Authorization token to access internal repo"
```
Comment (yejianquan) [inline .azure-pipelines/test_plan.py:348]: Recommend: Token to download the repo from Azure DevOps

File: .azure-pipelines/test_plan.py (hunk 1 above)
Comment (lerry-lee) [reply .azure-pipelines/test_plan.py]: fixed.

File: .azure-pipelines/test_plan.py (hunk 2 above)
Comment (lerry-lee) [reply .azure-pipelines/test_plan.py]: fixed.

**Review by yejianquan — APPROVED** [review]
LGTM

**Pull request merged**